examples/TicTacToe/scripts/deployTicTacToeContractsProxy.ts

```diff
@@ -1,15 +1,14 @@
 import { ethers } from "hardhat";
 import { Wallet, NonceManager, Signer } from "ethers";
 import { DeployUtils } from "@peer3/state-channels-plus";
-import dotenv from "dotenv";
 import {
     TicTacToeStateChannelManagerProxy,
     TicTacToeStateMachine
-} from "../typechain-types";
+} from "../../../typechain-types";
 import path from "path";
+import peer3Config from "../../../peer3config.json";
```

Comment:
As mentioned in the previous PR - this one isn't using peer3config it uses its own .env just to know where to deploy - even though it also has 'PROVIDER_URL' it's not logically the same provider as peer3config.PROVIDER_URL even though it's the same name and in most cases the same provider - so revert this

Reply:
It's fine if you want to change the example to use .json instead of .env for this, but then don't use the config in the sdk repo root '../../../' but have one in the example project root (where its own package.json is)